packages/core/src/modules/credentials/formats/CredentialFormatService.ts

```diff
@@ -35,7 +35,7 @@ export abstract class CredentialFormatService<CF extends CredentialFormat = Cred
 
   // proposal methods
   abstract createProposal(options: FormatCreateProposalOptions<CF>): Promise<FormatCreateProposalReturn>
-  abstract processProposal(options: FormatProcessOptions): Promise<void>
+  abstract processProposal(options: FormatProcessOptions): void
```

Comment:
this should stay async (it's an interface that could have different implementation in the future that may need async logic)